stdlib/2and3/ftplib.pyi

```diff
@@ -82,8 +82,8 @@ class FTP:
     # In practice, `rest` rest can actually be anything whose str() is an integer sequence, so to make it simple we allow integers.
     def ntransfercmd(self, cmd: Text, rest: Optional[_IntOrStr] = ...) -> Tuple[socket, int]: ...
     def transfercmd(self, cmd: Text, rest: Optional[_IntOrStr] = ...) -> socket: ...
-    def retrbinary(self, cmd: Text, callback: Callable[[bytes], Any], blocksize: int = ..., rest: Optional[_IntOrStr] = ...) -> str: ...
-    def storbinary(self, cmd: Text, fp: BinaryIO, blocksize: int = ..., callback: Optional[Callable[[bytes], Any]] = ..., rest: Optional[_IntOrStr] = ...) -> str: ...
+    def retrbinary(self, cmd: Text, callback: Callable[[Union[bytes, str]], Any], blocksize: int = ..., rest: Optional[_IntOrStr] = ...) -> str: ...
```

Comment:
This is problematic because it will now only accept callables whose argument type is `Union[bytes, str]`. I think you need to move the Union out, so it takes a Union of callables that take bytes and callables that take str.

Reply:
Ah I see, `Union[Callable[bytes, Any], Callable[str, Any]`?